src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/CreateRoom.java

```diff
@@ -64,16 +67,18 @@ public void react(ReceivedMessage parsedMessage, Messenger messenger, Database d
         DataDTO receivedMessage = (DataDTO) parsedMessage.getData();
 
         try {
-            validateData(receivedMessage);
+            validateData(receivedMessage, database);
 
-            User user = new User(receivedMessage.userDTO.id, receivedMessage.userDTO.name, this.connectionHashCode);
+            User user = new User(receivedMessage.userDTO.uuid, receivedMessage.userDTO.name, this.connectionHashCode);
             Room room = new Room(UUID.randomUUID(), receivedMessage.roomDTO.name, receivedMessage.roomDTO.password, user);
 
+            System.out.println("UUID pokoju: " + room.getUuid());
```

Comment:
I propose using static methods of `Log` class. You could write there something as `Log.DEBUG("UUID pokoju: " + room.getUuid()`, then this print would be blue on the terminal and simultaneously it'll be written to the log file. There are multiple levels of logs: `Log.TRACE`, `Log.DEBUG`, `Log.WARNING` and `Log.ERROR`.

Reply:
I accept the proposal